### PR TITLE
Enable simple snow e2e tests in CI

### PR DESCRIPTION
**Pull request by d8660091**
*Issue #, if available:* https://github.com/aws/eks-anywhere-internal/issues/1192

*Description of changes:* Enable simple snow e2e tests using the new snow CI devices

*Testing (if applicable):* Manually invoked the tests with codebuild, test was successful and verified that snow devices have been cleaned after testing.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: cmd/integration_test/build/buildspecs/snow-test-eks-a-cli.yml [hunk 1]
```diff
@@ -2,8 +2,8 @@ version: 0.2
 
 env:
   variables:
-    INTEGRATION_TEST_MAX_EC2_COUNT: 180
```
Comment (abhay-krishna) [inline cmd/integration_test/build/buildspecs/snow-test-eks-a-cli.yml:5]: Will this count suffice?

File: cmd/integration_test/build/buildspecs/snow-test-eks-a-cli.yml [hunk 2]
```diff
@@ -45,15 +49,16 @@ phases:
         -i ${INTEGRATION_TEST_INSTANCE_PROFILE}
         -m ${INTEGRATION_TEST_MAX_EC2_COUNT}
         -p ${INTEGRATION_TEST_MAX_CONCURRENT_TEST_COUNT}
-        -r 'Test'
```
Comment (abhay-krishna) [inline cmd/integration_test/build/buildspecs/snow-test-eks-a-cli.yml:48]: My vote is to do this filtering/skipping non-simpleflow tests through the SKIPPED_TESTS.yaml file while still keeping this regex as `Test`.

File: cmd/integration_test/build/buildspecs/snow-test-eks-a-cli.yml (hunk 1 above)
Comment (d8660091) [reply cmd/integration_test/build/buildspecs/snow-test-eks-a-cli.yml:5]: We have limited compute resource for only 3 snow devices, I will increase the number gradually in the subsequent PRs.

File: cmd/integration_test/build/buildspecs/snow-test-eks-a-cli.yml (hunk 2 above)
Comment (d8660091) [reply cmd/integration_test/build/buildspecs/snow-test-eks-a-cli.yml]: changed

**Pull request merged**